lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -82,7 +82,7 @@ def define
           case calculator.benefit
           when "jsa"
             if calculator.already_abroad && calculator.eea_country?
-              outcome :jsa_eea_already_abroad_outcome # A3 already_abroad
+              outcome :jsa_eea_already_abroad_outcome
```

Comment:
Hmmm nothing has changed here except for removal of a comment, which suggests to me that the tests being removed/rewritten should be moved to a more relevant (previous) commit.